next-packages/core/src/global.js

```diff
@@ -67,19 +67,19 @@ class InnerGlobal extends React.Component<InnerGlobalProps> {
       this.sheet.flush()
     }
 
-    rules.forEach(rule => {
-      this.sheet.insert(rule)
-    })
+    rules.forEach(this.sheet.insert, this.sheet)
   }
 
   componentWillUnmount() {
     this.sheet.flush()
   }
   render() {
     if (!isBrowser) {
-      const serialized = serializeStyles(this.props.context.registered, [
-        this.props.styles
-      ])
+      const serialized = serializeStyles(
```

Comment:
it seems to me that the most common use case is to pass `undefined` as first argument, maybe it would be better to reorder the arguments for this?